includes/class-newspack-popups-model.php

```diff
@@ -636,7 +640,8 @@ public static function get_access_attrs( $popup ) {
 		if (
 			( 'test' === $popup['options']['frequency'] || Newspack_Popups::previewed_popup_id() ) &&
 			is_user_logged_in() &&
-			current_user_can( 'edit_others_pages' )
+			current_user_can( 'edit_others_pages' ) ||
+			Newspack_Popups_Settings::is_non_interactive()
```

Comment:
Very minor nit, but this logic is pretty tough to untangle. Might be more readable to simply add a separate check-and-bail for `is_non_interactive()`.

Reply:
Done in 215e1a2